presto-docs/src/main/sphinx/installation/spark.rst

```diff
@@ -0,0 +1,70 @@
+=========================
+Executing Presto on Spark
+=========================
+
+When there is a need to execute long running or memory-intensive queries, Presto offers
+an option to scale the resource management by using Spark, as it doesn't offer
+one currently by itself. This way Presto can be executed as a standalone 
+application on Spark.
+
+Steps
+-----
+
+Download the Presto Spark package tarball, :maven_download:`spark-package` 
+and the Presto Spark laucher, :maven_download:`spark-launcher`. Keep both the
+files at, say, *example* directory. We assume here a two node Spark cluster
+with four cores each, thus giving us total eight cores.
+
+The following is an example ``config.properties``:
+
+.. code-block:: none
+
+    task.concurrency=4
+    task.max-worker-threads=4
+    task.writer-count=4
+     
+The details about properties are available at :doc:`/admin/properties`.
+Note that ``task.concurrency``, ``task.writer-count`` and 
+``task.max-worker-threads`` are set to 4 each, since we have four cores per executor
+and want to synchronize with the relevant Spark submit arguments below. 
+These values are to be adjusted to keep all executor cores busy and 
+synchronize with Spark submit parameters.
```

Comment:
Make `spark-submit` code and don't captialize

Reply:
Done